server/ai_http.go

```diff
@@ -106,6 +106,18 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 		ctx = clog.AddVal(ctx, "request_id", requestID)
 		ctx = clog.AddVal(ctx, "stream_id", streamID)
 
+		monitor.SendQueueEventAsync("stream_trace", map[string]interface{}{
+			"type":        "orchestrator_receive_stream_request",
+			"timestamp":   time.Now().UnixMilli(),
+			"stream_id":   streamID,
+			"request_id":  requestID,
+			"pipeline_id": "",
+			"orchestrator_info": map[string]interface{}{
+				"address": h.orchestrator.Address().Hex(),
+				"url":     h.orchestrator.ServiceURI().String(),
```

Comment:
We need the githash of go-livepeer (O) to be propagated here as well -- similarly a githash for the ai-runner in the runner events.